### ISSUE TITLE
responseType should be set to 'id_token'

### ISSUE THREAD

**Issue by chrischabot** [luisrudge/next.js-auth0#22]
The example uses JWT instead of plain access tokens, however utils/lock.js uses a 'token'  responseType, which leads to an invalid token error as there's no id_token to parse in utils/auth.js.

```
const getOptions = (container) => {
  const secret = uuid.v4()
  setSecret(secret)
  return {
    container,
    closable: false,
    auth: {
      responseType: 'token',
      redirectUrl: `${getBaseUrl()}/auth/signed-in`,
      params: {
        scope: 'openid profile email',
        state: secret
      }
    }
  }
}
```

Changing the responseType to  'id_token' (https://auth0.com/docs/tokens/id-token) fixes this issue:

```
const getOptions = (container) => {
  const secret = uuid.v4()
  setSecret(secret)
  return {
    container,
    closable: false,
    auth: {
      responseType: 'id_token',
      redirectUrl: `${getBaseUrl()}/auth/signed-in`,
      params: {
        scope: 'openid profile email',
        state: secret
      }
    }
  }
}
```

**Comment by luisrudge** [comment]
Thanks for pinging me. Can you send a PR with this?

**Comment by luisrudge** [comment]
Thanks! Closed with #23
